libs/os/os.go

```diff
@@ -21,7 +23,7 @@ func TrapSignal(logger logger, cb func()) {
 	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
 	go func() {
 		for sig := range c {
-			logger.Info(fmt.Sprintf("captured %v, exiting...", sig))
+			logger.Info("signal trapped", log.LazySprintf("captured %v, exiting...", sig))
```

Comment:
This produces the following log:

```
I[2022-06-23|12:18:29.328] signal trapped                               module=main capturedinterrupt,exiting...=(MISSING)
```

I think this is because the various `Logger` interface functions expect a message, followed by key/value pairs. Is it not possible to still keep the following call structure?

```suggestion
			logger.Info(log.LazySprintf("captured %v, exiting...", sig))
```

Alternatively I'd suggest something like:

```suggestion
			logger.Info("signal trapped", "msg", log.LazySprintf("captured %v, exiting...", sig))
```